lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1749,6 +1749,12 @@ definitions:
       gx_gy_relay_enabled:
         type: boolean
         example: false
+      subscriberdb_sync_interval:
```

Comment:
If a description is allowed, I would mention that the unit is seconds here. (Is it?)
Or just post pend the field with `_seconds`/`_secs`, even.

Reply:
Good point! Added description.